core/services/ardupilot_manager/settings.py

```diff
@@ -88,7 +89,30 @@ def save(self, content: Dict[str, Any]) -> None:
             Path.mkdir(self.settings_path, exist_ok=True)
 
             with open(self.settings_file, "w+") as file:
-                print(f"Updating settings file: {self.settings_file}")
+                logger.info(f"Updating settings file: {self.settings_file}")
                 json.dump(self.root, file, sort_keys=True, indent=4)
         except Exception as error:
-            warn(f"Could not save settings to disk: {error}")
+            logger.warning(f"Could not save settings to disk: {error}")
+
+
+log_config = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "default": {
+            "()": "uvicorn.logger.DefaultFormatter",
+            "fmt": "%(levelprefix)s %(asctime)s %(message)s",
+            "datefmt": "%Y-%m-%d %H:%M:%S",
+        },
+    },
+    "handlers": {
+        "default": {
+            "formatter": "default",
+            "class": "logger.StreamHandler",
+            "stream": "ext://sys.stderr",
```

Comment:
Is there a reason to push it to stderr and not stdout ?

Reply:
First thing: this settings.json change was not supposed to go ahead, so I'm removing that.

Despite that, the loguru default call specifies that, so it's still a valid discussion.

I basically have no strong reason, besides it being the default place that both [python](https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial) and [loguru](https://github.com/Delgan/loguru/blob/master/loguru/__init__.py) use as sink.